packages/terra-form-input/src/terra-dev-site/doc/form-input/Input.1.doc.mdx

```diff
@@ -24,6 +23,8 @@ import FormInputPropsTable from 'terra-form-input/src/Input?dev-site-props-table
 
 Generic input which represents an HTML input element directly.
 
+**Note:** To follow UX Best Practices, this component does not allow `placeholder`.
```

Comment:
At some point we should have a writeup about why we made this decision and link out to it here

Reply:
Added link to [w3 Placeholder Research](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Placeholder_Research) in `Note` section.